src/create-dashboard.js

```diff
@@ -63,34 +72,36 @@ module.exports = function (selector, diJSON, visOpts) {
 
   // TODO: We can probably move this logic somewhere else
   var widgetModels = [];
-  diJSON.widgets.forEach(function (widget) {
-    var widgetAttrs = _.omit(widget, 'dataview');
+  vizJSON.widgets.forEach(function (rawWidgetData) {
+    var layerId = rawWidgetData.layerId;
+    var widgetAttrs = _.omit(rawWidgetData, 'options', 'layerId');
+    var dataviewModel;
 
     // Create dataview if there's a definition provided
-    var dataviewModel;
-    var d = widget.dataview;
-    if (d) {
+    var dataviewAttrs = rawWidgetData.options;
+    if (dataviewAttrs) {
+      // TODO not ideal, should have a more maintainable way of mapping
+      dataviewAttrs.type = rawWidgetData.type === 'time-series'
+        ? 'histogram' // Time-series widget is represented by a histogram, so re-map the type
+        : rawWidgetData.type;
```

Comment:
@alonsogarciapablo one of the issues with changing the vizJSON data structure, since it no longer is a 1to1 match there needs to be some massaging of the data to map things as expected. Any thoughts on how we could do this better?